composer.json

```diff
@@ -96,11 +96,13 @@
         "roundcube/plugin-installer": "*"
     },
     "require": {
-        "composer-plugin-api": "^1.0"
+        "composer-plugin-api": "^1.0|^2.0"
     },
     "require-dev": {
-        "composer/composer": "1.0.*@dev",
-        "phpunit/phpunit": "^4.8.36"
+        "composer/composer": "1.0.0|2.0.*@dev",
```

Comment:
This + prefer latest/`--prefer-lowest` allows us to test the plugin against Composer 1 and Composer 2